### PR TITLE
INOV8-82 - add optional property 'regression_branch'

### PR DESCRIPTION
**Pull request by kevinp-q4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +78,7 @@ A string to represent the time when the TestRail run completed in the action.
   uses: q4mobile/stratus-testrail-reporter@v1
   with:
     target_branch: ${{ github.base_ref }}
+    regression_branch: test ## optional
```
Comment (vitalizatroutine) [inline README.md:81]: since the default is `staging`, perhaps readme should reflect

File: src/run.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import {
+  getInput,
+  setFailed,
+  getMultilineInput,
+} from "@actions/core";
+import { isEmpty } from "lodash";
+import moment from "moment-timezone";
+import TestRailApiClient, { IMilestone, INewTestRun } from "testrail-api";
+import { createTestPlan, createTestRun, extractError, getTestRailMilestone, readFiles } from "./utils";
+import {  } from "./utils/createTestRun";
```
Comment (vitalizatroutine) [inline src/run.ts:10]: ?

Comment (kevinp-q4) [reply src/run.ts:10]: oops removed

File: src/run.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+const environment = process.env.NODE_ENV || "debug";
+
+export enum InputKey {
+  RegressionBranch = "regression_branch",
+  TargetBranch = "target_branch",
+}
+
+export async function run(): Promise<void> {
+  const regressionBranch = getInput(InputKey.RegressionBranch) || "staging";
+  const regressionMode = getInput(InputKey.TargetBranch) === regressionBranch;
```
Comment (kevinp-q4) [inline src/run.ts:20]: this is the code I changed

File: README.md (hunk 1 above)
Comment (kevinp-q4) [reply README.md]: I added it above too but it doesn't hurt to have it here either

**Review by vitalizatroutine — APPROVED** [review]

**Pull request merged**